kernel/src/main/java/org/kframework/main/Main.java

```diff
@@ -38,6 +38,8 @@
 
 public class Main {
 
+    public static final long startTime = System.currentTimeMillis();
```

Comment:
this is not going to play nice with the k server. Can you please put this in a request-scoped value that is requested by the Main class that is instantiated (see the constructor on line 90) and then inject it into the Profiler2?